<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlanetExplorerDockWidgetBase</class>
 <widget class="QDockWidget" name="PlanetExplorerDockWidgetBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>431</width>
    <height>543</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Planet Explorer</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <property name="spacing">
     <number>6</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QStackedWidget" name="stckdWidgetViews">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="pageLogin">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QWidget" name="widgetLogin" native="true">
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="pixmap">
              <pixmap resource="../resources/resources.qrc">:/plugins/planet_explorer/planet-logo-dark.svg</pixmap>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Fixed</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>16</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLineEdit" name="leUser">
             <property name="text">
              <string/>
             </property>
             <property name="placeholderText">
              <string>Email address</string>
             </property>
             <property name="clearButtonEnabled">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QgsPasswordLineEdit" name="lePass">
             <property name="placeholderText">
              <string>Password</string>
             </property>
             <property name="clearButtonEnabled">
              <bool>true</bool>
             </property>
             <property name="showLockIcon" stdset="0">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QCheckBox" name="chkBxSaveCreds">
               <property name="toolTip">
                <string>Store credentials in QGIS authentication database</string>
               </property>
               <property name="text">
                <string>Save credentials</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDialogButtonBox" name="buttonBoxLogin">
               <property name="standardButtons">
                <set>QDialogButtonBox::Abort|QDialogButtonBox::Ok</set>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="verticalSpacer_3">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Fixed</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>12</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="lblTermsOfService">
             <property name="text">
              <string>By logging in you agree to the &lt;a href=&quot;https://learn.planet.com/QGIS-terms-conditions.html&quot; style=&quot;color: rgb(0, 157, 165);text-decoration: none;&quot;&gt;Terms and Conditions&lt;/a&gt;</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_4">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Fixed</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>12</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="lblSignUp">
             <property name="text">
              <string>Not a member? &lt;a href=&quot;https://www.planet.com/contact/&quot; style=&quot;color: rgb(0, 157, 165);text-decoration: none;&quot;&gt;Contact us!&lt;/a&gt;</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
             <property name="textInteractionFlags">
              <set>Qt::TextBrowserInteraction</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="lblForgotPass">
             <property name="text">
              <string>&lt;a href=&quot;https://www.planet.com/login?mode=reset-password&quot; style=&quot;color: rgb(0, 157, 165);text-decoration: none;&quot;&gt;Forgot password?&lt;/a&gt;</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
             <property name="textInteractionFlags">
              <set>Qt::TextBrowserInteraction</set>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="pageSearch">
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <property name="leftMargin">
         <number>9</number>
        </property>
        <property name="topMargin">
         <number>9</number>
        </property>
        <property name="rightMargin">
         <number>9</number>
        </property>
        <property name="bottomMargin">
         <number>9</number>
        </property>
        <item>
         <widget class="QFrame" name="frameSearch">
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <property name="spacing">
            <number>6</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="lblPLogo">
             <property name="maximumSize">
              <size>
               <width>36</width>
               <height>36</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="pixmap">
              <pixmap resource="../resources/resources.qrc">:/plugins/planet_explorer/planet-logo-p.svg</pixmap>
             </property>
             <property name="scaledContents">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>&lt;b&gt;Preview, Download or Stream Planet Imagery&lt;b/&gt;</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QTabWidget" name="tabWidgetResourceType">
          <property name="autoFillBackground">
           <bool>true</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">QTabWidget::tab-bar{alignment:center;}</string>
          </property>
          <property name="currentIndex">
           <number>1</number>
          </property>
          <widget class="QWidget" name="tabWidgetResourceTypePage1">
           <attribute name="title">
            <string>Daily Imagery</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>6</number>
            </property>
            <property name="topMargin">
             <number>6</number>
            </property>
            <property name="rightMargin">
             <number>6</number>
            </property>
            <property name="bottomMargin">
             <number>6</number>
            </property>
           </layout>
          </widget>
          <widget class="QWidget" name="tabWidgetResourceTypePage2">
           <attribute name="title">
            <string>Basemaps</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_9">
            <property name="leftMargin">
             <number>6</number>
            </property>
            <property name="topMargin">
             <number>6</number>
            </property>
            <property name="rightMargin">
             <number>6</number>
            </property>
            <property name="bottomMargin">
             <number>6</number>
            </property>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QgsMessageBar" name="msgBar"/>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsPasswordLineEdit</class>
   <extends>QLineEdit</extends>
   <header>qgspasswordlineedit.h</header>
  </customwidget>
  <customwidget>
   <class>QgsMessageBar</class>
   <extends>QFrame</extends>
   <header>qgis.gui</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
